.gitmodules

```diff
@@ -1,12 +1,14 @@
 [submodule "atmos_cubed_sphere"]
-	path = atmos_cubed_sphere
-	url = https://github.com/NOAA-EMC/GFDL_atmos_cubed_sphere
-	branch = dev/emc
+   path = atmos_cubed_sphere
```

Comment:
Please revert all whitespace/tab changes in `.gitmodules`

Reply:
done. Extra space and comment lines are removed.